SPECS/byacc/byacc.spec

```diff
@@ -53,6 +53,9 @@ echo ====================TESTING END=====================
 %{_mandir}/man1/byacc.1*
 
 %changelog
+* Thu Jan 25 2024 Bala <balakumaran.kannan@microsoft.com> - 1.9.20240109-1
```

Comment:
Can you please attach a Buddy Build Pipeline result.

Reply:
Buddy-Build is failing at toolchain download step. Will check later.

Reply:
BuddyBuild using 24/Jan toolkit. Latest daily repos are not stable. Also 24/Jan doens't have RPMs for ARM64. So, the pipeline ran successfully only for amd64 - https://dev.azure.com/mariner-org/mariner/_build/results?buildId=492896&view=logs&j=7909073b-e471-53c1-2c44-627e2efc0564&t=fbeef549-39ab-510c-faa3-78579a516b53